.gitignore

```diff
@@ -4,6 +4,3 @@ doc/tag/*
 
 # Ignore CMake build directory.
 build/
-
-# Ignore macOS metadata files.
-.DS_Store
```

Comment:
I think this should be left here, as we can still use Macs to develop even if we don't test on them.

Reply:
Might as well remove all references.

Reply:
I disagree since .DS_Store files are created by Finder accessing directories and are unrelated to the Mac port. It's a minor annoyance having them show up in git status, and a customer developing on a Mac might feel the same.

Reply:
Then this would be a problem in every git repo, right? You should probably add it to a global .gitignore: git config --global core.excludesfile ~/.gitignore